src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/PackageSourceTelemetry.cs

```diff
@@ -374,27 +390,35 @@ internal static Totals GetTotals(IReadOnlyDictionary<string, Data> data)
 
                     bytes += source.Value.NupkgSize;
                 }
+
+                if (source.Value.HttpSourceHasHttpResource)
+                {
+                    numberOfSourcesWithAnHttpResource++;
+                }
             }
 
-            return new Totals(requests, bytes, duration);
+            return new Totals(requests, bytes, duration, numberOfSourcesWithAnHttpResource);
         }
 
         public class Totals
         {
-            public Totals(int requests, long bytes, TimeSpan duration)
+            public Totals(int requests, long bytes, TimeSpan duration, int numberOfSourcesWithAnHttpResource)
             {
                 Requests = requests;
                 Bytes = bytes;
                 Duration = duration;
+                NumberOfSourcesWithAnHttpResource = numberOfSourcesWithAnHttpResource;
             }
 
             public int Requests { get; }
             public long Bytes { get; }
             public TimeSpan Duration { get; }
+            public int NumberOfSourcesWithAnHttpResource { get; }
         }
 
         internal class Data
         {
+            internal bool HttpSourceHasHttpResource { get; set; }
```

Comment:
Am I missing a subtly or should this also be HttpsSource?

```suggestion
            internal bool HttpsSourceHasHttpResource { get; set; }
```